arches/app/media/js/views/graph-designer.js

```diff
@@ -71,6 +85,62 @@ define([
                 window.open(arches.urls.export_mapping_file(viewModel.graph.graphid()), '_blank');
             };
 
+            viewModel.openPublishModal = function() {
```

Comment:
@chrabyrd we should replace these methods (`openPublishModal` and `closeModal`) with an observable and a `visible` binding on the `.modal-background` element.  We should always avoid doing DOM manipulation in viewmodels.

Reply:
now that I think about it... it might be fine (and maybe even nice) to use an observable boolean to drive the `visible` binding but retain these methods and modify them to simply toggle that observable value